### DISCUSSION TITLE
How to retrieve values of multiple selected options in HTML <select>?

### DISCUSSION THREAD

**Discussion by jamesrhester** [GenieFramework/Genie.jl#719] [Q&A]
As far as I can tell, if multiple options are selectable in an HTML form, then postpayload(:id) will only return the last value selected, instead of a list of selected values. What is the recommended way to work with multiple options in HTML <select> blocks?

**Comment by essenciary** [comment]
@jamesrhester If you have multiple inputs with the same name, you need to add the `[]` suffix to the name - ex `name="foo[]"`. These values will be collected into a vector.

**Comment by jamesrhester** [comment]
This doesn't seem to work. For the html (part of a form):
```html
<select id=instrs name=instrs[] multiple>
<option value=E>E</option>
<option value=W>W</option>
<option value=K>K</option>
<option value=P>P</option>
</select>
```
Submitting with the first two options selected and calling `print($(postpayload(:instrs)))` in `route` I get
`KeyError: key :instrs not found`

The relevant section of the raw payload is
```
Content-Disposition: form-data; name="instrs[]"

E
-----------------------------42394692521712607313909109619
Content-Disposition: form-data; name="instrs[]"

W
-----------------------------42394692521712607313909109619--

```
